bundles/mapping/mapmodule/plugin/layers/coveragetool/CoverageToolButton.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { ThemeConsumer } from 'oskari-ui/util';
+import { getNavigationTheme } from 'oskari-ui/theme';
+import { Button } from 'oskari-ui';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+const StyledButton = styled(Button)`
```

Comment:
Could we use this instead: https://github.com/oskariorg/oskari-frontend/blob/master/bundles/mapping/mapmodule/MapModuleButton.jsx?

Reply:
This won't work well when the content of the button is something else than an icon (=text). The text will overflow the tiny button and since additional styling would be necessary anyway then why not go all the way...